src/types/orchestrationTypes.ts

```diff
@@ -1,13 +1,18 @@
 import { FunctionOptions, FunctionTrigger, InvocationContext } from "@azure/functions";
-import { TokenSource } from ".";
+import { Task, TokenSource } from ".";
 import { DurableOrchestrationContext } from "../classes";
 
 /**
  * Type of a Generator that can be registered as an orchestration
+ * @param T the return value of the orchestration
  */
-export type OrchestrationHandler = (
+export type OrchestrationHandler<T = any> = (
     context: OrchestrationContext
-) => Generator<unknown, unknown, any>;
+) => Generator<
+    Task, // orchestrations yield Task types
+    T, // return type of the orchestration
```

Comment:
I feel like this should be `TReturn` to match the naming of `Generator`

Reply:
Done